build-system/start_sauce_connect.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+# This script starts the sauce connect proxy, and waits for a successful
+# connection.
+
+CYAN() { echo -e "\033[0;36m$1\033[0m"; }
+YELLOW() { echo -e "\033[1;33m$1\033[0m"; }
+
+DOWNLOAD_URL="https://saucelabs.com/downloads/sc-4.4.12-linux.tar.gz"
+TAR_FILE="sc-4.4.12-linux.tar.gz"
+BINARY_FILE="sc-4.4.12-linux/bin/sc"
```

Comment:
Consider hoisting the filename `sc-4.4.12-linux` to a separate var.

Reply:
Done.